lib/github_service/command_dispatcher.rb

```diff
@@ -26,6 +26,7 @@ def dispatch!(issuer:, text:)
       lines.each do |line|
         match = command_regex.match(line.strip)
         next unless match
+        next if issuer == GithubService.bot_name
```

Comment:
Interestingly we also have the bot_name method down below on line 55

Reply:
Should we coalesce around a single location for getting that?

Reply:
@Fryguy Alright, I made this consistent now.